### DISCUSSION TITLE
howto: split nextjs page to @pages lib package ?

### DISCUSSION THREAD

**Discussion by orenmizr** [nrwl/nx#7647] [General]
it is not imported - it is simply extracted by the nextjs bundler by looking at the pages/ folder. advice ?

**Comment by juristr** [comment]
By default next.js requires you to have the page components directly in the actual application. Meaning you have to locate the page components in the pages folder. however what you can do is to create a library and have a one-to-one mapping of a component there which you then import in the very thin layer of page components in the next application
